tests/spdx3/bump/test_package_bump.py

```diff
@@ -3,26 +3,36 @@
 # SPDX-License-Identifier: Apache-2.0
 from unittest import TestCase, mock
 
+import pytest
+
 from spdx_tools.spdx3.bump_from_spdx2.package import bump_package
 from spdx_tools.spdx3.model import ExternalIdentifier, ExternalIdentifierType, ExternalReference, ExternalReferenceType
 from spdx_tools.spdx3.model.software import Package
 from spdx_tools.spdx3.payload import Payload
 from spdx_tools.spdx.model.package import ExternalPackageRef, ExternalPackageRefCategory
 from spdx_tools.spdx.model.package import Package as Spdx2_Package
-from tests.spdx.fixtures import package_fixture
+from tests.spdx.fixtures import actor_fixture, package_fixture
 
 
+@pytest.mark.parametrize(
+    "originator, expected_originator",
+    [
+        (actor_fixture(name="originatorName"), "https://doc.namespace#SPDXRef-Actor-originatorName-some@mail.com"),
+        (None, None),
```

Comment:
Let's also insert the NOASSERTION case while we are it.

Reply:
Done